### PR TITLE
HasModels.modelNames and Storage.getModel with array

### PR DESCRIPTION
**Pull request by loppear**

### CODE REVIEW

File: src/HasModels.js [hunk 1]
```diff
@@ -19,12 +24,12 @@ export default class HasModels {
 
   /**
    * Override to define the model names to access
-   * @return {object} (model identifier: class attribute) pairs
-   * @example model_names() { 
-   * return {'user': 'User'}
+   * @return {array|object} Model identities to add to this.models, or object of {identity: name}
```
Comment (mjreich) [inline src/HasModels.js]: Why the object instead of a straight string?

File: src/index.js [hunk 2]
```diff
@@ -155,13 +155,16 @@ export default class Storage {
...
   /**
    * Request a model based on its identity (name)
-   * @param {string} id The identity of a registered model
-   * @return {Promise}  The model class
+   * @param {string|array} id The identity of a registered model, or array of identities
+   * @return {Promise}  The model class(es)
    * @example app.get('storage').getModel('user')
    */
   
   getModel (id) {
```
Comment (mjreich) [inline src/index.js]: So based on our latest conversation, we end up with `setModel` and `getModel`, correct?

Comment (loppear) [inline src/index.js]: I proposed `setModel` and `model` (don't prefix the common use), but really either way.

File: src/HasModels.js (hunk 1 above)
Comment (loppear) [inline src/HasModels.js]: Will update the example to just use a string, either is acceptable now (if you want to provide a short/alternate name you can return a mapping, normal would be array of strings)

**Pull request merged**